classes/local/store/object_client_base.php

```diff
@@ -89,25 +91,40 @@ public function generate_presigned_url($contenthash, $headers = array()) {
      */
     public function define_client_check() {
         global $OUTPUT;
+
+        // TODO use MDL check api admin setting if available ???
```

Comment:
TODO use https://tracker.moodle.org/browse/MDL-67898 if available, since it uses Ajax so page loading times are very quick